infra/generate_cfgs.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+import glob
+import os
+
+def make_cfgs(bench, data_dir):
+  cwd = os.getcwd()
+  path = f"{data_dir}/{bench}"
+  runmodes = os.listdir(path)
+  for mode in runmodes:
+    os.chdir(f"{path}/{mode}")
```

Comment:
Sorry I didn't catch this, but I think it's bad practice to change the working directory
Can we build the relative path everywhere instead?

Reply:
Then all the dot and png files would get created in the top-level dir, which is not what we want